spec/features/admin_can_view_order_details_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "rails_helper"
+require "factory_helper"
+
+feature "admin user" do
+  scenario "can view individual orders" do
+    category = Category.create(
+      name: "Plants",
+      description: "Carnivorous plants!"
+    )
+
+    plant1 = category.products.create(name: "Plant 1",
+                        description: "This is the description for plant 1",
+                        price:       19.99,
+                        image_url:   "plants/plant-2.jpg",
+                        status:      "active")
+    plant2 = category.products.create(name: "Plant 2",
+                        description: "This is the description for plant 2",
+                        price:       29.99,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.